docs/install/virtual_machine.md

```diff
@@ -175,10 +178,6 @@ plt.title('How fast do you want to go today?')
 show(plt)
 ``` 
 
+### R Extras
```

Comment:
ah, got it. hmm, I'm not sure - perhaps someone on https://jekyllrb.com/docs/permalinks/